### PR TITLE
Settings dialog

### PR DESCRIPTION
**Pull request by jaokob**
@kaszim will hook up the generate_conf method with the actual config generation.

### CODE REVIEW

**Comment by kaszim** [comment]
I've added cfg dumping. However you do not display the values already set to the user. @jaokob

**Comment by kaszim** [comment]
Also `Worst` and `Best` should be lowercase.

File: main.py [hunk 1]
```diff
@@ -132,6 +139,7 @@ def generate_conf(self):
         cfg[CONFIG_QUALITY] = quality
         cfg[CONFIG_BUFFER_SIZE] = buffer_size
 
+
```
Comment (kaszim) [inline main.py]: ?

**Comment by mictab** [comment]
@jaokob Can you resolve the conflicts? 😊

**Comment by jaokob** [comment]
@mictab sure, I'll fix them in a moment!

**Pull request merged**